settings.gradle

```diff
@@ -1,3 +1,19 @@
+plugins {
+  id 'com.gradle.develocity' version '3.17.5'
+}
+
+develocity {
+  buildScan {
+    termsOfUseUrl = 'https://gradle.com/terms-of-service'
+    termsOfUseAgree = 'yes'
```

Comment:
Accept only on CI?

Reply:
Won't matter, due to publish config below